src/USER-ADIOS/dump_atom_adios.h

```diff
@@ -0,0 +1,70 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef DUMP_CLASS
+
+DumpStyle(atom/adios,DumpAtomADIOS)
+
+#else
+
+#ifndef LMP_DUMP_ATOM_ADIOS_H
+#define LMP_DUMP_ATOM_ADIOS_H
+
+#include "dump_atom.h"
+#include <stdlib.h>
```

Comment:
please do not include headers except the ones required for parent classes into "style" headers. also, please use the c++ versions of c-library headers.